OUDS/Core/Components/Sources/Buttons/Internal/ButtonStyle+BackgroundModifier.swift

```diff
@@ -50,14 +54,16 @@ struct ButtonBackgroundModifier: ViewModifier {
         }
     }
 
-    private var enabledColor: MultipleColorSemanticTokens {
+    private var enabledColor: MultipleColorSemanticTokens? {
         switch hierarchy {
         case .default:
             useMonochrome ? theme.button.buttonMonoColorBgDefaultEnabled : theme.button.buttonColorBgDefaultEnabled
         case .strong:
             useMonochrome ? theme.button.buttonMonoColorBgStrongEnabled : theme.colors.colorActionEnabled
+        case .brand:
+            theme.button.buttonColorBgBrandEnabled
         case .minimal:
-            useMonochrome ? theme.button.buttonMonoColorBgMinimalEnabled : theme.button.buttonColorBgMinimalEnabled
+            nil // TODO: Remove button tokens
```

Comment:
@pylapp  should be remote in 0.15.0 token version ?

Reply:
Je n'ai pas compris la question 😅
La version 3.0 du bouton a été mis à disposition avant la version 1.5.0 de des tokens.
Si tu as pu implémenter le bouton conformément aux spécifications Figma au token près c'est que tu n'as pas forcément besoin de la v1.5.0 des tokens.

Par contre je ne sais pas à quoi tu fais référence en parlant de tokens "0.15.0".